app/src/org/commcare/activities/HomeButtons.java

```diff
@@ -27,8 +29,8 @@
  */
 public class HomeButtons {
 
-    private final static String[] buttonNames =
```

Comment:
Apologies, I should have provided more guidance here. Hiding the button can be accomplished more easily and without having to make additional modifications to this code. Instead, we just need to enhance StandardHomeActivityUIController.getHiddenButtons so it correctly indicates when to hide the Job Status button. The list of button names returned by that function is passed through to HomeButtons so buttons can be shown or hidden as indicated.